### PR TITLE
Fix route shapes

### PR DESCRIPTION
**Pull request by sr-vala**
Route shapes were a bit unrealistic after the infralink dump was updated.

Resolves HSLdevcom/jore4#1274

- - -
This change is [Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599)

### CODE REVIEW

**Review by jarkkoka — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599)* status: all files reviewed, 2 unresolved discussions (waiting on @sr-vala)

___
*[`cypress/e2e/createRoute.cy.ts` line 137 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599#-NW1sHhK-aiKbNTVB4t3:-NW1sHhK-aiKbNTVB4t4:buoanqg) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/4c84c9beb29c479417b500497a67a57b61edb1a5/cypress/e2e/createRoute.cy.ts#L137)):*
> ```XML
>     infrastructure_link_id: infrastructureLinkIds[2],
>     infrastructure_link_sequence: 2,
>     is_traversal_forwards: true,
> ```

This seems like a no-op change since you are just ordering array objects differently (from ascending to descending in terms of `infrastructure_link_sequence`). Makes not much sense to me.
___
*[`cypress/e2e/editRouteShape.cy.ts` line 126 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599#-NW1rm-B-7mRL4DCDKiE:-NW1rm-C5DnCq3d2Pics:buoanqg) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/4c84c9beb29c479417b500497a67a57b61edb1a5/cypress/e2e/editRouteShape.cy.ts#L126)):*
> ```XML
>   {
>     route_id: routes[0].route_id,
>     infrastructure_link_id: infrastructureLinkIds[2],
> ```

This seems like a no-op change since you are just ordering array objects differently (from ascending to descending in terms of `infrastructure_link_sequence`). Makes not much sense to me.

**Review by jarkkoka — CHANGES_REQUESTED** [review]
> *[Reviewable](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599)* status: all files reviewed, 2 unresolved discussions (waiting on @sr-vala)

___
*[`cypress/e2e/createRoute.cy.ts` line 137 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599#-NW1sHhK-aiKbNTVB4t3:-NW1snS84gl79ucBQaf8:bfabld2) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/4c84c9beb29c479417b500497a67a57b61edb1a5/cypress/e2e/createRoute.cy.ts#L137)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

This seems like a no-op change since you are just ordering array objects differently (from ascending to descending in terms of `infrastructure_link_sequence`). Makes not much sense to me.
</blockquote></details>

In other words, the route shape does not change.
___
*[`cypress/e2e/editRouteShape.cy.ts` line 126 at r2](https://reviewable.io/reviews/HSLdevcom/jore4-ui/599#-NW1rm-B-7mRL4DCDKiE:-NW1srCQ9dMQXwpA1hOX:bfabld2) ([raw file](https://github.com/HSLdevcom/jore4-ui/blob/4c84c9beb29c479417b500497a67a57b61edb1a5/cypress/e2e/editRouteShape.cy.ts#L126)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

This seems like a no-op change since you are just ordering array objects differently (from ascending to descending in terms of `infrastructure_link_sequence`). Makes not much sense to me.
</blockquote></details>

In other words, the route shape does not change.
